README.md

```diff
@@ -846,7 +846,7 @@ And this set of workflow options:
 
 ```json
 {
-  "defaultRuntimeOptions": {
+  "default_runtime_options": {
```

Comment:
If we're taking the chance to change this, the choice of the word `options` has always annoyed me. Should probably be `default_runtime_attributes` instead.

(ps it was me who originally named this, mea culpa)